cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -637,12 +637,11 @@ def alternative_identifiers(self, key, value):
             raise IgnoreKey("alternative_identifiers")
         # conference_info.identifiers mixed data
         if sub_9.upper() == "INSPIRE-CNUM":
-            _conference_info = self.get("conference_info", {})
-            _prev_identifiers = _conference_info.get("identifiers", [])
```

Comment:
are you sure about this change ? with the new implementation you are just replacing any previous identifiers